src/components/vuestic-components/va-icon/VaIcon.vue

```diff
@@ -59,7 +63,7 @@ export default {
       return (this.icon && this.icon.component) || this.c_component || this.c_tag
     },
     computedClass () {
-      return (this.icon && this.icon.iconClass) || ''
+      return this.icon ? this.icon.iconClass : this.spin ? 'material-icons spin' : ''
```

Comment:
Why would you add `material-icons` here? This should work with any iconfont.

Reply:
Different icon fonts working differently. Awesome working by adding CSS class, Material - by adding text in `i`. How I suppose to detect which font is used?